src/Invoice/Inv/Service/PlaywrightPdfRenderService.php

```
@@ -0,0 +1,62 @@
+        ];
+        // Points at node_modules/playwright-core/.local-browsers (installed
+        // via `PLAYWRIGHT_BROWSERS_PATH=0 npx playwright install chromium`)
+        // instead of the interactive user's profile — Apache's service
+        // account has its own separate profile and can't see the latter.
+        $process = new Process($args, $root, ['PLAYWRIGHT_BROWSERS_PATH' => '0']);
+        $process->setTimeout(120);
+        $process->run();
+        if (!$process->isSuccessful()) {
+            throw new PlaywrightRenderFailedException($translator, $process->getErrorOutput());
+        }
+    }
```

PHPMD (2.15.0)

[error] render accesses the super-global variable $_ENV. (undefined)

(Superglobals)